src/react-medium-image-zoom.js

```diff
@@ -157,6 +159,7 @@ export default class ImageZoom extends Component {
   handleZoom(event) {
     if (this.props.shouldHandleZoom(event)) {
       this.setState({ isZoomed: true })
+      this.props.onZoom()
```

Comment:
Good point. For my use it wouldn't matter, but this is definitely safer generally.